src/SrBot.ts

```diff
@@ -197,12 +199,54 @@ export class SrBot {
                 this.sendToChannel(message.channel, `Team name: ${server && server.teamName || 'NOT SET'}`);
                 break;
             }
+            case COMMAND.ADD: {
+                const username = params[0];
+                if (this.isValidUsername(username)){
+                    this.addPlayer(message, username);
+                    this.sendToChannel(message.channel, `Added ${username} to your roster`);
+                } else {
+                    this.sendToChannel(message.channel, `Invalid username: ${username}`);
+                }
+                break;
+            }
+            case COMMAND.REMOVE: {
+                const username = params[0];
+                if (this.isValidUsername(username)){
+                    this.removePlayer(message, username);
+                    this.sendToChannel(message.channel, `Removed ${username} to your roster`);
+                } else {
+                    this.sendToChannel(message.channel, `Invalid username: ${username}`);
+                }
+                break;
+            }
             default:
                 // command not found
                 break;
         }
     }
 
+    private addPlayer(message : Discord.Message, username: string) : boolean {
+        const serverId = message.member.guild.id;
+        return this.statsGenerator.addPlayer(serverId, username);
+    }
+
+    private removePlayer(message : Discord.Message, username : string) : boolean {
```

Comment:
Are these `addPlayer`/`removePlayer` methods needed in this class?  Could we call just call `this.statsGenerator.###Player(serverId, username);` where they are used?

Reply:
Done